packages/aws-cdk-lib/aws-cloudfront/lib/distribution.ts

```diff
@@ -129,7 +129,10 @@ export interface DistributionProps {
    *
    * If you want to use your own domain name, such as www.example.com, instead of the cloudfront.net domain name,
    * you can add an alternate domain name to your distribution. If you attach a certificate to the distribution,
-   * you must add (at least one of) the domain names of the certificate to this list.
+   * you should add (at least one of) the domain names of the certificate to this list.
+   *
+   * When you want to move a domain name between distributions, you can associate a certificate without specifying any domain names.
+   * See [Moving an alternate domain name to a different distribution](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/CNAMEs.html#alternate-domain-names-move) for details.
```

Comment:
Reading the docs, I see that moving the domain name requires calling the `associate-alias` CLI command. So this change just allows the user to create a target distribution but they would still need to call `associate-alias` correct?  I think it would be helpful to include an example of how someone would do this in CDK in the README.

Reply:
Yes. We should call `associate-alias` cli command manually.
I will add the instructions.

Reply:
I've added a new section to the README.
Thanks for your suggestion!